internal/services/containers/kubernetes_nodepool.go

```diff
@@ -365,10 +366,16 @@ func schemaNodePoolKubeletConfig() *pluginsdk.Schema {
 			},
 		},
 	}
+
+	if !features.FourPointOhBeta() {
+		schema.Elem.(*pluginsdk.Resource).Schema["cpu_cfs_quota_enabled"].Default = false
+	}
```

Comment:
Can we revert the logic here to set the `Default` to `true` when it is 4.0, and adding a commnet above the `if` as a TODO, reminding the maintainer to add the default value to the document when updating to 4.0?

(same as below)

Reply:
Got it, I've added comment.